desktop/dashboard.md

```diff
@@ -220,3 +224,21 @@ To interact with remote repositories:
     The **View in Hub** option opens the Docker Hub page and displays detailed information about the image, such as the OS architecture, size of the image, the date when the image was pushed, and a list of the image layers.
 
     If you have subscribed to a Pro or a Team plan and have enabled [Vulnerability Scanning](../docker-hub/vulnerability-scanning.md) the Docker Hub page also displays a summary of the vulnerability scan report and provides detailed information about the vulnerabilities identified.
+
+## Manage volumes
```

Comment:
The previous section is `Explore your images`, should this one be `Manage your volumes`?

Reply:
Added "Manage Volumes" as a heading to be consistent with "Volume Management" in the blog draft.

Reply:
What I meant is the TOC looks a bit inconsistent

![image](https://user-images.githubusercontent.com/2386884/120988341-c3d11b80-c77e-11eb-89a3-0374534fd2ef.png)

Reply:
Flipped the order of the headings.